kolibri/plugins/epub_viewer/assets/src/views/SearchSideBar.vue

```diff
@@ -169,6 +169,12 @@
           color: this.$themeTokens.annotation,
         };
       },
+      markColor() {
+        return {
+          color: this.$themePalette.black,
+          backgroundColor: this.$themePalette.yellow.v_a200,
```

Comment:
Seems like typo, I think it should be `yellow.v_200`

Reply:
Fixed